Examples/PcapPrinter/main.cpp

```diff
@@ -82,6 +82,8 @@ std::string linkLayerToString(pcpp::LinkLayerType linkLayer)
 
 	if (linkLayer == pcpp::LINKTYPE_ETHERNET)
 		return "Ethernet";
+	if (linkLayer == pcpp::LINKTYPE_IEEE802_5)
+		return "LINKTYPE_IEEE802_5";

```

Comment:
A more readable string might be: `IEEE 802.5 Token Ring`

Reply:
Resolved in 27c7d04